.github/dependabot.yml

```diff
@@ -0,0 +1,69 @@
+version: 2
+updates:
+  - package-ecosystem: "gomod"
+    directory: "/"
+    schedule:
+      interval: "daily"
+    allow:
+      - dependency-type: "all"
+    reviewers:
```

Comment:
Same comment as on SSP: Do we need to set it at all? Won't kubevirt-bot assign one of the reviewers? IIRC I was assigned to previous PRs of dependabot.

Reply:
But e.g. I wasnt and I would like to know about the PRs.

Reply:
But won't this add all of the reviewers everytime? If you want to be notified about every PR on the repo, can't you subscribe to it individually instead?